packages/compat/lib/tracks/client.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @deprecated 7.5.0 use Automattic\Jetpack\Tracking->tracks_record_event_raw instead
+ */
+function jetpack_tracks_record_event_raw( $event ) {
```

Comment:
should we `_deprecate_function()` these as well?

Reply:
yesss

Reply:
https://github.com/Automattic/jetpack/pull/12907/commits/2907733029a3da15930bc5dab1ae9fa40a463624